src/components/Table/Table.hooks.js

```diff
@@ -24,6 +26,7 @@ export function useTable(data = [], maxRowsToDisplay = null) {
   const [state, dispatch] = useReducer(reducer, {
     selectedRows: [],
     currentTableData: initialDisplayTableData,
+    columns: [...columns],
```

Comment:
curious about why we need to create a new array instance?

Reply:
I had issues at some point with 😈 mutation 😈 on the `columns` (when toggling the "show" flag, but I need the original columns untouched for the "reset to defaults" option)